lib/mina/kubernetes.rb

```diff
@@ -18,6 +18,13 @@
     apply_kubernetes_resources(args[:options])
   end
 
+  task :global_deploy, [:options] do |task, args|
+    set_tag_from_branch_commit unless fetch(:image_tag)
+    wait_until_image_ready(fetch(:image_tag)) unless fetch(:skip_image_ready_check)
+    create_namespace_on_cluster
```

Comment:
Is this one still needed if it's deploying to the global namespace?

Reply:
i wouldn't think so, thanks!